go/teams/box_audit.go

```diff
@@ -671,27 +671,27 @@ var _ libkb.TeamBoxAuditor = &DummyBoxAuditor{}
 const dummyMsg = "Box auditor disabled; aborting successfully"
 
 func (d DummyBoxAuditor) AssertUnjailedOrReaudit(mctx libkb.MetaContext, _ keybase1.TeamID) (bool, error) {
-	mctx.Warning(dummyMsg)
+	mctx.Debug(dummyMsg)
```

Comment:
There is a warning when the box auditor is installed onto G if it's a dummy. (happens in bot lite mode)